tests/activityAPIRequest.spec.js

```diff
@@ -0,0 +1,112 @@
+const { test, expect } = require('@playwright/test');
+const { createRecord, getList, updateRecord, deleteRecord } = require('../helpers/requestMethods.js');
+const { EndPoints } = require('../Constants/EndPoint.js');
+const { Data } = require('../test-data/payloadBody.js');
+const RandomDataGenerator = require('../helpers/randomHelpers.js');
+const RandomDateGenerator = require('../helpers/dateHelper.js');
+
+
+const endPoints = EndPoints();
+
+test('Validate the activity list getting successfully', async ({ request }) => {
+    const response = await getList(request, endPoints.activities);
+
+    expect(await response.status()).toBe(200);
+    expect(await response.ok()).toBeTruthy();
+});
+
+test('Validate the Activity is cretaed succsessfully', async ({ request }) => {
+    const postData = Data.activities;
+
+    const postAPIResponse = await createRecord(request, postData, endPoints.activities);
+
+    expect(await postAPIResponse.status()).toBe(200);
+    expect(await postAPIResponse.ok()).toBeTruthy();
+
+    const postAPIResponseBody = await postAPIResponse.json();
+
+    expect(postAPIResponseBody).toHaveProperty("id", postData.id);
+    expect(postAPIResponseBody).toHaveProperty("title", postData.title);
+    expect(postAPIResponseBody).toHaveProperty("dueDate", postData.dueDate);
+});
+
+test('Validate the activity is not cretaed succsessfully', async ({ request }) => {
+    const testeDatas = [
```

Comment:
can move this test data into separate class and then use it here

Reply:
Done